src/SqlServer/test/Eventuous.Tests.SqlServer/Subscriptions/SubscriptionFixture.cs

```diff
@@ -33,29 +34,36 @@ protected override SqlServerCheckpointStore GetCheckpointStore(IServiceProvider
         => new(sp.GetRequiredService<SqlServerCheckpointStoreOptions>());
 
     protected override void ConfigureSubscription(TSubscriptionOptions options) {
-        options.Schema           = _schemaName;
+        options.Schema           = SchemaName;
         options.ConnectionString = Container.GetConnectionString();
         configureOptions(options);
     }
 
     protected override void SetupServices(IServiceCollection services) {
         base.SetupServices(services);
-        services.AddSingleton(new SchemaInfo(_schemaName));
-        services.AddEventuousSqlServer(Container.GetConnectionString(), _schemaName, true);
+        services.AddSingleton(new SchemaInfo(SchemaName));
+        services.AddEventuousSqlServer(Container.GetConnectionString(), SchemaName, true);
         services.AddAggregateStore<SqlServerStore>();
-        services.AddSingleton(new SqlServerCheckpointStoreOptions { Schema = _schemaName, ConnectionString = Container.GetConnectionString() });
+        services.AddSingleton(new SqlServerCheckpointStoreOptions { Schema = SchemaName, ConnectionString = Container.GetConnectionString() });
         services.AddSingleton(new TestEventHandlerOptions(null, _outputHelper));
         configureServices?.Invoke(services);
     }
 
+    protected override void GetDependencies(IServiceProvider provider) {
+        base.GetDependencies(provider);
+        ConnectionString = Container.GetConnectionString();
```

Comment:
This line isn't hit because `base.GetDependencies(provider);` failes. I don't understand why.
The result is that the ProjectorTest tests are failing. All other tests work.

Reply:
Stacktrace:

```
CallSiteFactory.CreateArgumentCallSites(ServiceIdentifier serviceIdentifier, Type implementationType, CallSiteChain callSiteChain, ParameterInfo[] parameters, Boolean throwIfCallSiteNotFound)
CallSiteFactory.CreateConstructorCallSite(ResultCache lifetime, ServiceIdentifier serviceIdentifier, Type implementationType, CallSiteChain callSiteChain)
CallSiteFactory.TryCreateExact(ServiceDescriptor descriptor, ServiceIdentifier serviceIdentifier, CallSiteChain callSiteChain, Int32 slot)
CallSiteFactory.TryCreateExact(ServiceIdentifier serviceIdentifier, CallSiteChain callSiteChain)
CallSiteFactory.CreateCallSite(ServiceIdentifier serviceIdentifier, CallSiteChain callSiteChain)
CallSiteFactory.GetCallSite(ServiceIdentifier serviceIdentifier, CallSiteChain callSiteChain)
ServiceProvider.CreateServiceAccessor(ServiceIdentifier serviceIdentifier)
ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
ServiceProviderEngineScope.GetService(Type serviceType)
<16 more frames...>
ServiceProvider.CreateServiceAccessor(ServiceIdentifier serviceIdentifier)
ConcurrentDictionary`2.GetOrAdd(TKey key, Func`2 valueFactory)
ServiceProvider.GetService(ServiceIdentifier serviceIdentifier, ServiceProviderEngineScope serviceProviderEngineScope)
ServiceProvider.GetService(Type serviceType)
ServiceProviderServiceExtensions.GetRequiredService(IServiceProvider provider, Type serviceType)
ServiceProviderServiceExtensions.GetRequiredService[T](IServiceProvider provider)
SubscriptionFixtureBase`5.GetDependencies(IServiceProvider provider) line 63
SubscriptionFixture`3.GetDependencies(IServiceProvider provider) line 53
StoreFixtureBase`1.InitializeAsync() line 47
SubscriptionFixtureBase`5.InitializeAsync() line 74
```